server/channels/app/email/email_batching.go

```diff
@@ -376,3 +393,26 @@ func (es *Service) sendBatchedEmailNotification(userID string, notifications []*
 		mlog.Warn("Unable to send batched email notification", mlog.String("email", user.Email), mlog.Err(nErr))
 	}
 }
+
+func (es *Service) GenerateHyperlinkForChannels(teamId, postMessage, landingURL string) (string, error) {
```

Comment:
To avoid duplicating `App.generateHyperlinkForChannels` here, could you make it so that that method calls this one? That's the same thing we do for `App.GetMessageForNotification` already

Reply:
Actually, see my other comment. If we have `GetMessageForNotification` call this, then we can just remove `App.generateHyperlinkForChannels` instead of duplicating it

Reply:
Removed duplicates. Needed some refactoring on codes and tests for this